recipes-wpe/wpeframework/wpeframework_git.bb

```diff
@@ -121,6 +121,10 @@ EXTRA_OECMAKE += " \
     -DEXCEPTIONS_ENABLE=${WPEFRAMEWORK_EXCEPTIONS_ENABLE} \
 "
 
+SELECTED_OPTIMIZATION_append = "\
+    ${@bb.utils.contains('PACKAGECONFIG', 'debugoptimized', " -O0 " , '', d)} \
```

Comment:
Shouldn't the optimization flags be handled at CMake level? Does the buildroot build bahaves the same way?

Reply:
Let me check

Reply:
I moved this to Thunder So will close this PR